app/scripts/contentscript.js

```diff
@@ -13,6 +13,13 @@ const repoName = window.location.pathname
 var content = saveClass("new-discussion-timeline experiment-repo-nav")
 var repoContent = saveClass("repository-content")
 let metrics = []
+var chaves = ""
```

Comment:
Please keep the standardized code in English. Replace 'chaves' to 'keys' for example.

Reply:
Ok, the variable name no is 'popup_key'

Reply:
seems solved